### PR TITLE
fix: precision issue

### PR DESCRIPTION
**Pull request by Mikelle**
## Describe your changes

Fixing precision issue

## Issue ticket number and link

Fixes https://github.com/primev/mev-commit/issues/492

## Checklist before requesting a review

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: contracts/contracts/core/BidderRegistryStorage.sol [hunk 1]
```diff
@@ -9,14 +9,13 @@ abstract contract BidderRegistryStorage {
     using FeePayout for FeePayout.Tracker;
 
     /// @dev For improved precision
-    uint256 constant public PRECISION = 1e25;
-    uint256 constant public PERCENT = 100 * PRECISION;
+    uint256 constant public PERCENT = 1e18;
```
Comment (ckartik) [inline contracts/contracts/core/BidderRegistryStorage.sol:12]: Why is percent 1e18?

Comment (Mikelle) [reply contracts/contracts/core/BidderRegistryStorage.sol:12]: To fit into int64 in the oracle decay calculation

File: contracts/scripts/core/DeployCore.s.sol [hunk 2]
```diff
@@ -29,8 +29,8 @@ contract DeployTestnet is Script {
         address protocolFeeRecipient = address(
             0xfA0B0f5d298d28EFE4d35641724141ef19C05684 // Placeholder for now, L1 preconf.eth address
         );
-        uint16 feePercent = 2;
-        uint16 providerPenaltyPercent = 5;
+        uint256 feePercent = 2 * 1e16; // 2%
```
Comment (mrekucci) [inline contracts/scripts/core/DeployCore.s.sol:32]: The number `1e16` repeats, consider creating a constant.

File: contracts/test/core/BidderRegistryTest.sol [hunk 3]
```diff
@@ -194,7 +194,7 @@ contract BidderRegistryTest is Test {
 
         bidderRegistry.openBid(commitmentDigest, 1 ether, bidder, blockNumber);
 
-        bidderRegistry.retrieveFunds(nextWindow, commitmentDigest, payable(provider), 100);
+        bidderRegistry.retrieveFunds(nextWindow, commitmentDigest, payable(provider), 1e18);
```
Comment (mrekucci) [inline contracts/test/core/BidderRegistryTest.sol:197]: Shouldn't be the `1e18` replaced with `PERCENT` constant?

File: contracts/test/core/BidderRegistryTest.sol [hunk 4]
```diff
@@ -269,7 +269,7 @@ contract BidderRegistryTest is Test {
         vm.expectRevert(bytes(""));
         bytes32 commitmentDigest = keccak256("1234");
         bidderRegistry.openBid(commitmentDigest, 1 ether, bidder, blockNumber);
-        bidderRegistry.retrieveFunds(nextWindow, commitmentDigest, payable(provider),100);
+        bidderRegistry.retrieveFunds(nextWindow, commitmentDigest, payable(provider),1e18);
```
Comment (mrekucci) [inline contracts/test/core/BidderRegistryTest.sol:272]: The same as above.

File: contracts/test/core/BidderRegistryTest.sol [hunk 5]
```diff
@@ -287,7 +287,7 @@ contract BidderRegistryTest is Test {
         vm.prank(address(this));
         bytes32 commitmentDigest = keccak256("1234");
         bidderRegistry.openBid(commitmentDigest, 3 ether, bidder, blockNumber);
-        bidderRegistry.retrieveFunds(nextWindow, commitmentDigest, payable(provider),100);
+        bidderRegistry.retrieveFunds(nextWindow, commitmentDigest, payable(provider),1e18);
```
Comment (mrekucci) [inline contracts/test/core/BidderRegistryTest.sol:290]: The same is here and also in several other places below.

**Review by ckartik — APPROVED** [review]

**Pull request merged**